src/sitemap.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think it would be helpful to have a comment explaining that this file is referenced from robots.txt and serves the purpose of helping robots know what to index (and how often, I gather). If there are any better places to go to learn about what should be here than the namespace urls, it would be helpful to mention some, too.